pandas-stubs/io/gbq.pyi

```diff
@@ -12,10 +12,10 @@ def read_gbq(
     dialect: str | None = ...,
     location: str | None = ...,
     configuration: dict[str, Any] | None = ...,
-    credentials=...,
+    # Google type, not available
+    credentials: Any = ...,
     use_bqstorage_api: bool | None = ...,
-    private_key=...,
-    verbose=...,
+    max_results: int | None = ...,
     progress_bar_type: str | None = ...,
```

Comment:
`progress_bar_type` can be `Literal["tqdm", "tqdm_notebook", "tqdm_gui"]` according to docs